app/Policies/CustomMapPolicy.php

```diff
@@ -45,7 +45,7 @@ public function create(User $user): bool
      */
     public function update(User $user, CustomMap $customMap): bool
     {
-        return $user->hasGlobalRead() || $customMap->hasAccess();
+        return false;
```

Comment:
Shouldn't this be $user->hasAdmin()?

Reply:
There is a function at the top of the file that I'm guessing runs before all requests, and this does the hasAdmin() check and returns true.

Reply:
This is the function at the top of the file:
 
    public function before(User $user): ?bool
    {
        if ($user->isAdmin()) {
            return true;
        }

        return null;
    }

Reply:
Ah, didn't see that in the diff :)